tests/db_functions/math/test_cot.py

```diff
@@ -0,0 +1,44 @@
+import math
+from decimal import Decimal
+
+from django.db.models import DecimalField
+from django.db.models.functions import Cot
+from django.test import TestCase
+
+from ..models import DecimalModel, FloatModel, IntegerModel
+
+
+class CotTests(TestCase):
+
+    def test_decimal(self):
+        DecimalModel.objects.create(n1=Decimal('-12.9'), n2=Decimal('0.6'))
+        obj = DecimalModel.objects.annotate(n1_cot=Cot('n1'), n2_cot=Cot('n2')).first()
+        self.assertAlmostEqual(obj.n1_cot, 1 / math.tan(obj.n1))
+        self.assertAlmostEqual(obj.n2_cot, 1 / math.tan(obj.n2))
+
+    def test_float(self):
+        FloatModel.objects.create(f1=-27.5, f2=0.33)
+        obj = FloatModel.objects.annotate(f1_cot=Cot('f1'), f2_cot=Cot('f2')).first()
+        self.assertAlmostEqual(obj.f1_cot, 1 / math.tan(obj.f1))
+        self.assertAlmostEqual(obj.f2_cot, 1 / math.tan(obj.f2))
+
+    def test_integer(self):
+        IntegerModel.objects.create(small=-5, normal=15, big=-1)
+        obj = IntegerModel.objects.annotate(
+            small_cot=Cot('small'),
+            normal_cot=Cot('normal'),
+            big_cot=Cot('big'),
+        ).first()
+        self.assertAlmostEqual(float(obj.small_cot), 1 / math.tan(obj.small))
+        self.assertAlmostEqual(float(obj.normal_cot), 1 / math.tan(obj.normal))
+        self.assertAlmostEqual(float(obj.big_cot), 1 / math.tan(obj.big))
```

Comment:
Remove the cast to `float` for these - it shouldn't be necessary due to `output_field = FloatField()`.